### PR TITLE
Peep renames broken

### PR DESCRIPTION
**Pull request by janisozaur**
1. Start a game
2. Try renaming a peep
3. See the assert getting hit.

### CODE REVIEW

**Comment by janisozaur** [comment]
May be related to https://github.com/OpenRCT2/OpenRCT2/issues/4670

**Comment by Gymnasiast** [comment]
Personally I would put the expressions in either side of the `<` between brackets, just to make sure it does what you mean.

**Comment by janisozaur** [comment]
@Gymnasiast relational operators always have lower precedence than unary and binary arithmetic operators. They have higher precedence than unary and binary logical and (non-)equal operators.

File: src/peep/peep.c [hunk 1]
```diff
@@ -12306,6 +12306,7 @@ money32 set_peep_name(int flags, int state, uint16 sprite_index, uint8* text_1,
 	static char newName[128];
 	//if (flags & GAME_COMMAND_FLAG_APPLY) { // this check seems to be useless and causes problems in multiplayer
 		uint8 position = (state - 1) % 3;
```
Comment (duncanspumpkin) [inline src/peep/peep.c]: Why dont you just fix the issue? `&3` instead of `%3`

**Pull request closed without merging**